### PR TITLE
fix-ELEMENTS-868-PropertiesNotBeingForwarded

### PR DESCRIPTION
**Pull request by nmpcunha**
https://qa.nuxeo.org/jenkins/view/TestAndPush/job/web-ui-pipeline-ncunha/103/

### CODE REVIEW

**Comment by guirenard** [comment]
:+1:

**Comment by Gabez0r** [comment]
👍

File: test/nuxeo-filter.html [hunk 1]
```diff
@@ -321,6 +382,58 @@
...
+      document.getElementById('mySlotContent').create();
+      return flush().then(() => {
+        const stampedElements = stamped(customElementsAndBehaviorsContainer, '.custom');
+        expect(stampedElements.length).to.be.equal(1);
+        let myElement = stampedElements[0];
+        expect(stampedOnShadowRoot(myElement, '#label').length).to.be.equal(1);
+        expect(myElement.label).to.be.equal('A simple test');
+        const slot = stamped(customElementsAndBehaviorsContainer, 'nuxeo-slot')[0];
+        slot.set('model', {document: {type: 'File'}, text: 'A simple test'});
+        return flush().then(() => {
```
Comment (Gabez0r) [inline test/nuxeo-filter.html]: You should be able to just return the flush here and do the `then` one level up, thus preventing promise nesting.

File: test/nuxeo-filter.html [hunk 2]
```diff
@@ -321,6 +382,58 @@
...
+          expect(stampedOnShadowRoot(myElement, '#label').length).to.be.equal(1);
+          slot.set('model', {document: {type: 'Folder'}, text: 'A simple test'});
+          return flush().then(() => {
```
Comment (Gabez0r) [inline test/nuxeo-filter.html]: same here

File: test/nuxeo-filter.html [hunk 3]
```diff
@@ -321,6 +382,58 @@
...
+      });
+    });
+    test('text should be undefined', () => {
+      const filter = Polymer.dom(domHostData).querySelector('nuxeo-filter');
+      filter.set('__dataHost', {});
+      filter.set('document', {type: 'File'});
+      return flush().then(() => {
+        expect(stamped(domHostData, '.text').length).to.be.equal(0);
+        filter.set('document', {type: 'Folder'});
+        return flush().then(() => {
```
Comment (Gabez0r) [inline test/nuxeo-filter.html]: same here

**Pull request merged**